src/node/transaction.cpp

```diff
@@ -78,6 +78,9 @@ TransactionError BroadcastTransaction(NodeContext& node, const CTransactionRef t
     }
 
     if (relay) {
+        // the mempool explicitly keeps track of wallet txns to ensure successful initial broadcast
```

Comment:
This also adds anything submitted via `sendrawtransaction`, at least, so it's not explicitly wallet-related.

I think it's wallet-or-sendrawtransaction transactions only :)

confirmed:
`src/node/transaction.cpp:19:    // BroadcastTransaction can be called by either sendrawtransaction RPC or wallet RPCs.`

this should get a test!

Reply:
Yup. Totally. I think I use the language of “local” transactions elsewhere. Do you find that clear or should I just spell it all out every time? 

Also, revisiting my `mempool_wallet_transactions` test, I realized I actually end up creating the “wallet” transaction with `sendrawtransaction`. Oops. I'll add another transaction in that test that uses `sendtoaddress` then it will cover both :) 

Thanks !

Reply:
{node, locally}-submitted transactions is fine, just noting the comments are wrong in certain places

Reply:
fixed